apps/docs/src/guide/wallets/wallet-transferring.md

```diff
@@ -1,14 +1,59 @@
 # Wallet Transferring
```

Comment:
I'm unsure how we should document `Account`-related methods. A predicate can transfer assets the same way as a wallet. Should we put it in some other section? Create a new `Accounts` section?
- Accounts
  - Transferring assets
  - Checking balances
  - etc.